structopt-derive/src/lib.rs

```diff
@@ -14,16 +14,30 @@ extern crate proc_macro;
 extern crate syn;
 #[macro_use]
 extern crate quote;
+extern crate heck;
 extern crate proc_macro2;
 
 mod attrs;
 
-use attrs::{Attrs, Kind, Parser, Ty};
+use attrs::{Attrs, CasingStyle, Kind, Parser, Ty};
 use proc_macro2::{Span, TokenStream};
 use syn::punctuated::Punctuated;
 use syn::token::Comma;
 use syn::*;
 
+/// Default casing style for generated arguments.
+const DEFAULT_CASING: CasingStyle = CasingStyle::Kebab;
```

Comment:
I can do that, but you should be aware, that changing the default into verbatim will also rename all kinds of existing commands with non defined names.

Here is a snippet i have taken from the `flatten_enum` example. Subcommands defined like this will no longer be callable by `test foo` but will change into `test Foo`.

```rust
    #[derive(StructOpt, Debug, PartialEq)]
    struct Opt {
        #[structopt(flatten)]
        sub_cmd: SubCmd,
    }
    #[derive(StructOpt, Debug, PartialEq)]
    enum SubCmd {
        Foo,
        Bar,
    }
```

Reply:
Mmm, that is the actual behavior, no?

Reply:
Right, I just checked it again and realized afterwards that I missed to compare against latest master instead of my branch before I wrote this comment.